pcdsdevices/at2k4_tmo.py

```diff
@@ -0,0 +1,55 @@
+"TMO sample calibration foil to work with spectrometer"
+from lightpath import LightpathState
+from ophyd.device import Component as Cpt
+
+from .device import GroupDevice
+from .device import UpdateComponent as UpCpt
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface, LightpathMixin
+from .pmps import TwinCATStatePMPS
+
+
+class CalibrationAxe(TwinCATStatePMPS):
+    """
+    Sample calibration 1D State Setup
+
+    Here, we specify 5 states, and 1 motor, for Y
+    axe.
+    """
+    config = UpCpt(state_count=5, motor_count=1)
+
+
+class SCaliFoil(BaseInterface, GroupDevice, LightpathMixin):
+    """
+    TMO PMPS1D sample calibration class.
+
+
+    Parameters:
+    -----------
+    prefix : str
+        Base PV for the motion system
+
+    name : str
+        Alias for the device
+    """
+    # UI Representation
+    _icon = 'fa.minus-square'
+    tab_component_names = True
+
+    # Sample calibration Y axe
+    at2k4 = Cpt(CalibrationAxe, 'AT2K4:IP1:STATE', add_prefix=(), kind='normal')
```

Comment:
The subcomponent names are confusing to me. `at2k4` is a component of `at2k4`?

Reply:
let me change the name here. I will talk to scientists about the name.

Reply:
The main reason to change these is so the states row doesn't say `at2k4 at2k4` when we add the device to happi.